AltV.Atlas.Boilerplate.Client/appsettings.json

```diff
@@ -1,4 +1,12 @@
 {
+  "TrafficSettings": {
+    "MaxTrafficVehiclesInStreamDistance": 30,
+    "MinimumTrafficNodesToGenerate": 10,
+    "SpawnRadius": 160,
+    "MinimumSpawnDistanceFromPlayer": 80,
+    "CleanupIntervalMs": 60000,
+    "TrafficSpawnIntervalMs": 1000
+  },
```

Comment:
```suggestion
```

As this is part of a premium module, it doesn't need to be in appsettings by default.